examples/generator-fullstack/model/datamodel.graphql

```diff
@@ -1,21 +1,22 @@
 type Note {
   id: ID!
+  """
+  @db.type: 'string'
+  @db.length: 100
+  """
   title: String!
-  description: String!
-  ## Relationship
-  comment: [Comment!]!
+  """
+  @db.type: 'text'
```

Comment:
We should have 'text' as default IMHO

Reply:
In Postgres there is no difference between TEXT and VARCHAR. However, there can be a difference in others, such as MySQL. In MySQL a TEXT column cannot be indexed. Since it is possible to customise, should we leave the default as VARCHAR?

Reply:
So those types are knex types or native types?

Reply:
Knex types.